about.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <!-- fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700|Gothic A1" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"
+        integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/fontawesome.min.css"
+        integrity="sha512-siarrzI1u3pCqFG2LEzi87McrBmq6Tp7juVsdmGY1Dr8Saw+ZBAzDzrGwX3vgxX1NkioYNCFOVC0GpDPss10zQ=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- style sheet  -->
+    <link rel="stylesheet" href="./css/about.css" />
+</head>
+ 
+<body>
+    <section class="container">
+        <header>
+            <nav class="socialnavbar">
+                <ul class="socialnav">
+                    <li>
+                        <a href="/"><i class="fa-brands fa-facebook-f"></i></a>
+                    </li>
+                    <li>
+                        <a href="/"><i class="fa-brands fa-twitter"></i></a>
+                    </li>
+                    <li><a href="/">english</a></li>
+                    <li><a href="/">my page</a></li>
+                    <li><a href="/">logout</a></li>
+                </ul>
+            </nav>
+
+            <nav class="navbar">
+                <div class="logo">
+                    <a href="/index.html"><img src="./image/logo.png" alt="" /></a>
+                </div>
+                <ul class="nav">
```

Comment:
Great work on your project and for taking the time to incorporate the changes suggested by the previous reviewer! I'd like to offer some optional suggestions that could potentially enhance your project even further:

- [ ] **[OPTIONAL]** If possible, could you please ensure that your page remains responsive without any layout issues when viewed at 800px width? This would make the user experience even better.
- [ ] **[OPTIONAL]** Additionally, on larger screens with resolutions like 1920x1080, it would be great if you could ensure that the page elements are perfectly aligned.
 
   For a better understanding of what I'm referring to, you can check out the video below.
   
   [scrnli_9_8_2023_19-39-23.webm](https://github.com/abdullah97-pro/capstoneModule1/assets/107952724/3ccb89d7-086e-43f2-a5bf-0f8e811458cc)

Reply:
Yes, thank you i will do